worlds/sm64ex/docs/en_Super Mario 64 EX Advance Setup.md

```diff
@@ -0,0 +1,66 @@
+# Super Mario 64 EX Advanced Setup Guide
+
+## Required Software
+
+- Super Mario 64 US Rom (Japanese will work also. Europe and Shindou not supported)

```

Comment:
The US and Japanese roms work equally well, but this use of parentheses implies US is preferred.
I suggest "Super Mario 64 Rom, either US or Japanese (Europe and Shindou are not supported)."